### PR TITLE
destructure parameters

### PR DESCRIPTION
**Pull request by ConeDragon**
https://forum.exercism.org/t/js-elyses-destructured-enchantments-exemplar-inconsistent/12084

### CODE REVIEW

**Comment by ConeDragon** [comment]
In exercise notes, should there be a note that destructuring in the function itself is also acceptable?

**Comment by SleeplessByte** [comment]
> In exercise notes, should there be a note that destructuring in the function itself is also acceptable?

That would be good. If you mentor this exercise it's a good opportunity to inform a student that in JavaScript you can do it directly in the parameter.

File: exercises/concept/elyses-destructured-enchantments/.meta/exemplar.js [hunk 1]
```diff
@@ -46,9 +42,7 @@ export function swapTopTwoCards([a, b, ...rest]) {
  * @returns {[Card, Card[]]} the top card of the given
  * deck and a new deck containing all the other cards
  */
-export function discardTopCard(deck) {
-  const [first, ...rest] = deck;
-
+export function discardTopCard([first, ..rest]) {
```
Comment (SleeplessByte) [inline exercises/concept/elyses-destructured-enchantments/.meta/exemplar.js:45]: Missing a `.` as it should be `...`

**Review by SleeplessByte — CHANGES_REQUESTED** [review]

File: exercises/concept/elyses-destructured-enchantments/.meta/exemplar.js (hunk 1 above)
Comment (ConeDragon) [reply exercises/concept/elyses-destructured-enchantments/.meta/exemplar.js:45]: Whoops! Sorry

**Review by SleeplessByte — APPROVED** [review]
Thank you!

**Pull request merged**